### PR TITLE
Computer Vision v3.1

### PR DESCRIPTION
**Pull request by areddish**
Computer Vision 3.1 Release. 
 - No API surface area changes
 - Improvements to the backend model for /describe (and /analyze when asking for Description) which may produce different results from v3.0

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithUrl.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "language": "en",
+    "ImageUrl": {
+      "url": "{url}"
+    }
+  },
+  "responses": {
+    "202": {
+      "header": {
+        "Operation-Location": "https://{domain}/vision/v3.0/read/e56ffa6e-1ee4-4042-bc07-993db706c95f"
```
Comment (ryogok) [inline specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithUrl.json:13]: Need to replace with v3.1?

File: specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithStream.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "language": "en",
+    "Image": "{binary}"
+  },
+  "responses": {
+    "202": {
+      "header": {
+        "location": "https://{domain}/vision/v3.0/read/e56ffa6e-1ee4-4042-bc07-993db706c95f"
```
Comment (ryogok) [inline specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithStream.json:11]: Need to replace with v3.1?

Comment (areddish) [reply specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithStream.json]: Replaced, thanks!

File: specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithUrl.json (hunk 1 above)
Comment (areddish) [reply specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/examples/SuccessfulReadWithUrl.json]: Replaced, thanks!

**Review by ryogok — APPROVED** [review]

**Review by TFR258 — APPROVED** [review]

**Review by lmazuel — APPROVED** [review]

**Pull request merged**